src/test/java/org/cactoos/io/HeadInputStreamTest.java

```diff
@@ -91,10 +90,10 @@ public void testAvailableLessThanTotal() throws Exception {
             new InputOf("testAvailableLessThanTotal").stream(),
             5
         );
-        MatcherAssert.assertThat(
-            "Count of available bytes is incorrect",
+        new Assertion<>(
+            "Must count available bytes",
```

Comment:
@victornoel minor comment: should we always start with a lower case letter for consistency? `must count available bytes`